conans/test/functional/command/test_command_test.py

```diff
@@ -30,3 +33,10 @@ def test_conan_test(self):
         subfolder = "Release" if platform.system() != "Windows" else ""
         assert os.path.exists(os.path.join(client.current_folder, "test_package",
                                            "build", subfolder, "generators", "conaninfo.txt"))
+
+    def test_whateverrr(self):
+        tc = TestClient()
+        tc.run("new hello/1.0 -m=cmake_lib")
```

Comment:
Do not use a full blown test, that compiles, and it is very slow, just to test folders.
Should be moved to ``integration``, and being tested with some simple recipe that do not build anything.